Tests/NIOPosixTests/AsyncChannelBootstrapTests.swift

```diff
@@ -656,6 +656,80 @@ final class AsyncChannelBootstrapTests: XCTestCase {
         }
     }
 
+    func testPipeBootstrap_whenInputNil() async throws {
+        let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        let (pipe1ReadFH, pipe1WriteFH) = self.makePipeFileDescriptors()
+        let fromChannel = FileHandle(fileDescriptor: pipe1ReadFH, closeOnDealloc: false)
+        let channel: NIOAsyncChannel<ByteBuffer, ByteBuffer>
+
+        do {
+            channel = try await NIOPipeBootstrap(group: eventLoopGroup)
+                .takingOwnershipOfDescriptors(
+                    input: nil,
+                    output: pipe1WriteFH
+                ) { channel in
+                    channel.eventLoop.makeCompletedFuture {
+                        try NIOAsyncChannel(synchronouslyWrapping: channel)
+                    }
+                }
+        } catch {
+            [pipe1ReadFH, pipe1WriteFH].forEach { try? SystemCalls.close(descriptor: $0) }
```

Comment:
```suggestion
            [pipe1ReadFH, pipe1WriteFH].forEach { try! SystemCalls.close(descriptor: $0) }
```

This must be a `!` or else we may miss NIO doing bad stuff.

Reply:
(alternatively just a `try` works too)